firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -151,11 +184,43 @@ bool moveAlongEdges(int startCol, int startRow, int endCol, int endRow)
   currCol = currPositionX / (stepsPerUnitSpace * 8);
   currRow = currPositionY / (stepsPerUnitSpace * 8);
 
+  // Print debug info about moveAlongEdges
+  if (DEBUG)
+  {
+    Serial.println("The following is in the form (col, row)");
+    Serial.print("Moving to starting position from (");
+    Serial.print(currCol);
+    Serial.print(", ");
+    Serial.print(currRow);
+    Serial.print(") to (");
+    Serial.print(startCol);
+    Serial.print(", ");
+    Serial.print(startRow);
+    Serial.print(") ");
+    Serial.println("\n");
+  }
+
   // Make initial move to first position. Move diagonally since it's faster.
   // Since we're moving to the initial position, we want the magnet off, so pass in false.
   if (!moveDirect(currCol, currRow, startCol, startRow, false))
     return false;
 
+  // Print debug info about stright movement
```

Comment:
i think this whole block needs to be updated to say moving along edges to … and the comment should be updated as well